ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/services/grpc/GrpcHealthService.scala

```diff
@@ -41,7 +40,7 @@ class GrpcHealthService(
   private val errorLogger: ContextualizedErrorLogger =
     new DamlContextualizedErrorLogger(logger, loggingContext, None)
 
-  import ErrorFactories.invalidArgument
+  import com.daml.ledger.api.validation.ValidationErrors.invalidArgument
```

Comment:
maybe importing it on the top section would read more nicely?

Reply:
Ok, done